README.rst

```diff
@@ -93,4 +80,5 @@ Credits
 
 * `Ger van Diepen <gervandiepen@gmail.com>`_
 * `Malte Marquarding <Malte.Marquarding@gmail.com>`_
+* `Tammo Jan Dijkema <dijkema@astron.nl>`_
```

Comment:
Why didn't you add yourself at the end?

Reply:
Didn't think about the order, indeed at the end is more correct.